src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -109,11 +118,16 @@ class FileField extends React.Component {
           uiOptions.isFileEncrypted || this.isFileEncrypted;
         const needsPassword = await isFileEncrypted(currentFile);
         if (needsPassword) {
-          files[idx] = {
-            file: currentFile,
-            name: currentFile.name,
-            isEncrypted: true,
-          };
+          if (enableShortWorkflow) {
+            files[idx].isEncrypted = true;
+          } else {
+            files[idx] = {
+              file: currentFile,
+              name: currentFile.name,
+              isEncrypted: true,
+            };
+          }
```

Comment:
Ended up removing this change, since it doesn't need guarding.  The only time we need the 'file' object is on resending and the action.js function takes care of sending the file back anyways.